addons/l10n_co/data/account_tax_template.xml

```diff
@@ -151,18 +154,22 @@
         <field name="tax_group_id" ref="tax_group_r_iva_075"/>
         <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {'repartition_type': 'base'}),
-            (0,0, {'repartition_type': 'tax'}),
+            (0,0, {
+                'repartition_type': 'tax',
+                'account_id': ref('co_puc_236700''),
```

Comment:
@aliyatastemirova I did the same error over here, can you help me to remove the additional '.

Thanks.